Drv/Ip/IpSocket.cpp

```diff
@@ -0,0 +1,201 @@
+// ======================================================================
+// \title  IpSocket.cpp
+// \author mstarch
+// \brief  cpp file for IpSocket core implementation classes
+//
+// \copyright
+// Copyright 2009-2020, by the California Institute of Technology.
+// ALL RIGHTS RESERVED.  United States Government Sponsorship
+// acknowledged.
+//
+// ======================================================================
+
+#include <Drv/Ip/IpSocket.hpp>
+#include <Fw/Types/Assert.hpp>
+#include <Fw/Types/BasicTypes.hpp>
+#include <Fw/Types/StringUtils.hpp>
```

Comment:
[Fw/Types/CMakeLists.txt](https://github.com/nasa/fprime/blob/update/ground-prototype/Fw/Types/CMakeLists.txt) needs to be updated with following lines:

https://github.com/nasa/fprime/blob/97883e50982a1275383a721f6c8a1c4c2aa1433e/Fw/Types/CMakeLists.txt#L20

https://github.com/nasa/fprime/blob/97883e50982a1275383a721f6c8a1c4c2aa1433e/Fw/Types/CMakeLists.txt#L35-L38